docs/examples/longhorn.md

```diff
@@ -0,0 +1,241 @@
+<!--
+SPDX-FileCopyrightText: 2025 k0s authors
+SPDX-License-Identifier: CC-BY-SA-4.0
+-->
+
+# Longhorn
+
+This tutorial covers installing Longhorn as a Helm extension on k0s. Longhorn is a distributed block storage system for Kubernetes that provides enterprise-grade features including:
+
+- **Distributed storage**: Provides persistent storage across your cluster nodes
+- **Snapshots and backups**: Built-in snapshot and backup capabilities for data protection
+- **Live upgrades**: Zero-downtime upgrades of storage volumes
+- **High availability**: Automatic failover and replication for data durability
+- **Cross-cluster disaster recovery**: Backup to external storage systems
+
+Longhorn is particularly well-suited for stateful applications that require persistent storage, such as databases, message queues, and content management systems.
+
+For more information on Longhorn, refer to the [official documentation](https://longhorn.io/docs/).
+
+## Prerequisites
+
+Before installing Longhorn, ensure your k0s cluster meets these requirements:
+
+### System Requirements
+
+- **Operating System**: Ubuntu 24.04+, SLES 15 SP7+, SLE Micro 6.1+, RHEL 10.0+, Oracle Linux 9.6+, Rocky Linux 10.0+, Talos 1.10.6+, Container-Optimized OS 121+
+- **Kernel**: Linux kernel 3.10 or later
+- **Storage**: At least 10GB of free disk space per node for Longhorn data
+- **Memory**: Minimum 2GB RAM per node (4GB+ recommended for production)
+- **Network**: Stable network connectivity between all cluster nodes
+
+For more detailed system requirements, refer to the [official Longhorn v1.10.x Support Matrix](https://www.suse.com/suse-longhorn/support-matrix/all-supported-versions/longhorn-v1-10-x/).
```

Comment:
Maybe the link to the upstream support matrix is enough, so we don't need to sync this list with upstream whenever they change something.